pkg/providers/amifamily/ami.go

```diff
@@ -219,11 +219,22 @@ func (p *Provider) fetchAMIsFromEC2(ctx context.Context, amiSelector map[string]
 	p.ec2Cache.SetDefault(fmt.Sprint(hash), output.Images)
 	amiIDs := lo.Map(output.Images, func(ami *ec2.Image, _ int) string { return *ami.ImageId })
 	if p.cm.HasChanged("amiIDs", amiIDs) {
-		logging.FromContext(ctx).With("ami-ids", amiIDs).Debugf("discovered images")
+		logging.FromContext(ctx).With("ami-ids", concatenateAmiIDs(amiIDs)).Debugf("discovered images")
 	}
 	return output.Images, nil
 }
 
+func concatenateAmiIDs(amisIds []string) string {
+	var amisIdsString string
+	if len(amisIds) > 5 {
+		amisIdsString = strings.Join(amisIds[:5], ", ")
+		amisIdsString += fmt.Sprintf(", (%d more amis)", len(amisIds)-5)
```

Comment:
Can we align the formatting of this with [instanceTypeList](https://github.com/aws/karpenter-core/pull/273/files#diff-f0cdb8c0527c28b9b19283810ab0deadc411c48d0f66ef2255109dd7824f7e7eR356)